lib/grape/validations/validators/default.rb

```diff
@@ -6,8 +6,30 @@ def initialize(attrs, options, required, scope)
         super
       end
 
```

Comment:
`private def` works fine for me.  It causes just that method to be marked private without altering the visibility of subsequent methods.

````ruby
$ irb
irb(main):001:0> class Foo
irb(main):002:1>   private def bar
irb(main):003:2>     'bar'
irb(main):004:2>   end
irb(main):005:1>   def baz
irb(main):006:2>     'baz'
irb(main):007:2>   end
irb(main):008:1> end
=> :baz
irb(main):009:0> foo = Foo.new
=> #<Foo:0x007fcae281d800>
irb(main):010:0> foo.baz
=> "baz"
irb(main):011:0> foo.bar
NoMethodError: private method `bar' called for #<Foo:0x007fcae281d800>
````
But I'm happy to break it up if that's what you'd prefer.